File: app/src/main/java/nju/se/simpletiktok/VideoInfo.java
package nju.se.simpletiktok;

import com.google.gson.annotations.SerializedName;

public class VideoInfo{
    @SerializedName("_id")
    public String id;
    @SerializedName("feedurl")
    public String videouri;
    @SerializedName("avatar")
    public String avatururi;
    @SerializedName("nickname")
    public String nickname;
    @SerializedName("description")
    public String description;
    @SerializedName("likecount")
    public int likecounts;
    @SerializedName("thumbnails")
    public String thumbnails;
}

File: app/src/main/java/nju/se/simpletiktok/VideoInfo.kt
package nju.se.simpletiktok

import com.google.gson.annotations.SerializedName

class VideoInfo {
    @SerializedName("_id")
    var id: String = ""

    @SerializedName("feedurl")
    var videouri: String = ""

    @SerializedName("avatar")
    var avatururi: String = ""

    @SerializedName("nickname")
    var nickname: String = ""

    @SerializedName("description")
    var description: String = ""

    @SerializedName("likecount")
    var likecounts = 0

    @SerializedName("thumbnails")
    var thumbnails: String = ""
}